include/simpleble/PeripheralSafe.h

```diff
@@ -13,19 +13,19 @@ class Peripheral : public SimpleBLE::Peripheral {
     Peripheral(SimpleBLE::Peripheral& peripheral);
     virtual ~Peripheral() = default;
 
-    std::optional<std::string> identifier() noexcept;
-    std::optional<BluetoothAddress> address() noexcept;
-    std::optional<int16_t> rssi() noexcept;
+    std::optional<std::string> identifier() const noexcept;
+    std::optional<BluetoothAddress> address() const noexcept;
+    std::optional<int16_t> rssi() const noexcept;
 
     bool connect() noexcept;
     bool disconnect() noexcept;
```

Comment:
Shouldn't these two also be `const`? Considering that these external classes only forward calls to the internal objects.

Reply:
Well yes it could but if you go this way then the whole PeripheralSafe class is a nonsense.

It's a class with only public methods, I would make a lot more sense for it to be an interface and then all the implementations for the os (linux/PeripheralSafe macos/PeripheralSafe windows/PeripheralSafe) will implement everything.

Same goes for Peripheral class